barretenberg/cpp/pil/avm/avm_main.pil

```diff
@@ -373,8 +375,12 @@ namespace avm_main(256);
 
     //===== CONTROL FLOW =======================================================
     //===== JUMP ===============================================================
+    #[PC_JUMP]
```

Comment:
i suppose these could potentially be combined into the same relation? sel_jump feels like a subset of jumpi

Reply:
Yes it is. In addition one can combine with:

-  sel_internal_call * (pc' - ia) = 0;
-  sel_internal_return * (pc' - ia) = 0;

I suggest to favor readability for the time being and I will write a comment in PIL file about this consolidation.